specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json

```diff
@@ -1450,20 +2063,43 @@
       "required": true,
       "type": "string"
     },
+    "resourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "description": "Azure Resource Group Name.",
+      "required": true,
```

Comment:
@bgsky - This parameter should also have `"x-ms-parameter-location": "method"`. Autorest by default turns global parameters into client properties. It is not a good idea to have resourceGroupName as a client property.

Reply:
There is no reason to put them globally. Would you add "x-ms-parameter-location": "method" to resourceGroupName and budgetName parameters quickly? Otherwise, I can update them.

Reply:
We will make this change in PR https://github.com/Azure/azure-rest-api-specs/pull/2386